lib/reference/__tests__/atKeywords.test.mjs

```diff
@@ -0,0 +1,9 @@
+import { atKeywords, nestingSupportedAtKeywords } from '../atKeywords.js';
+
+describe('atKeywords', () => {
+	it('nestingSupportedAtKeywords is a subset of all atKeywords', () => {
+		for (const keyword of nestingSupportedAtKeywords) {
+			expect(atKeywords).toContain(keyword);
+		}
+	});
+});
```

Comment:
minor

That would be the first test for the reference folder.
I think just by using `uniteSets` at l29 you can avoid adding it.
i.e. be explicit in the code

Reply:
> But atKeywords must always contain all known at keywords.
Even if a utility grouping is altered or removed.

Then you could run into the scenario that an at keyword was only added in a subset and then later removed from that subset without being added to the full set.

This is easy to spot when there is only one sub set, but would be harder to notice when there are several.

-------

Adding it to `uniteSets` would be explicit and it is how I initially wrote it.
But it wouldn't be obvious why there is a subset being added to the whole list when each item in the subset already exists in the whole list.

I don't mind changing it back :)

Reply:
I meant

```js
const atKeywords = uniteSets(pageMarginAtKeywords, [
	'annotation',
	'apply',
	'character-variant',
	'charset',
	'counter-style',
	'custom-media',
	'custom-selector',
	'document',
	'font-face',
	'font-feature-values',
	'import',
	'keyframes',
	'namespace',
	'nest',
	'ornaments',
	'page',
	'property',
	'scroll-timeline',
	'styleset',
	'stylistic',
	'swash',
	'viewport',
], nestingSupportedAtKeywords);
```